### PR TITLE
redesign of get started page

### PR DESCRIPTION
**Pull request by mirho**
Purpose of this redesign is to make it is easier to follow the instructions. This page is used by both Earthly users, and Earthly CI users, and will be linked to from within the CI as well.

- New design, with some more descriptive text labels in various places where there were none previously.
- Add one-click copy command to installation instructions for Linux, Mac, Windows, and Verify install instruction.
- Bring this page in line with our visual style
- Add link to Try Earthly CI, and Earthly CI docs link
- Change home page link text from 'Download Earthly' to 'Install Earthly'

image

### CODE REVIEW

**Review by thtmnisamnstr — APPROVED** [review]
I like it. Approved on my end.

File: website/get-earthly.md [hunk 1]
```diff
@@ -141,95 +157,97 @@ ide:
...
+  <div class="mt-2">
+See the <a href="https://docs.earthly.dev/earthly-cloud/earthly-ci" class="underline blue-link  font-semibold">Earthly CI docs</a>
+</div>
+
+<a href="/signup/earthly-ci" class="try-button py-4 h-10 px-4 xl:px-6 items-center text-sm xl:text-base text-center text-white bg-[#2d7e5d] hover:bg-green-800 rounded-lg inline-flex">
+                    Try Earthly CI
+                </a>
+
+  <div class="text-2xl mt-6 border-t pt-4 border-gray-300">
+    Other CI integrations
```
Comment (vladaionescu) [inline website/get-earthly.md:230]:
```suggestion
    Run Earthly in a traditional CI
```

File: website/get-earthly.md [hunk 2]
```diff
@@ -141,95 +157,97 @@ ide:
...
+  </div>
+<div class="mt-2">
+See the <a href="https://docs.earthly.dev/guides/ci-integration" class="underline  font-semibold blue-link">CI integration guide</a>
```
Comment (vladaionescu) [inline website/get-earthly.md:233]:
```suggestion
See the <a href="https://docs.earthly.dev/guides/ci-integration" class="underline  font-semibold blue-link">CI integration guide</a> to use Earthly in GitHub Actions, Circle CI, Jenkins, GitLab CI, and others.
```

**Review by vladaionescu — APPROVED** [review]
Looks good - just some minor suggestions regarding wording

**Pull request merged**